jdk/src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c

```diff
@@ -1,3 +1,25 @@
+/*******************************************************************************
+ * (c) Copyright IBM Corp. 2018, 2018 All Rights Reserved
+ *
+ * This program and the accompanying materials are made available under
```

Comment:
The license for the OpenJDK code cannot be changed, @pdbain-ibm please remove this license.

Reply:
The IBM licence was added on the advice of @andrew-m-leonard and follows
the practice in other IBM-modified files, e.g. java.base/share/classes/java/io/ObjectInputStream.java, java.base/share/classes/java/net/URLClassLoader.java, et al.

Reply:
These files add an IBM copyright but do not add an Eclipse or Apache license. Andrew has already fixed the license problem.